_studio/shared/umc/codec/mpeg2_dec/src/umc_mpeg2_dec.cpp

```diff
@@ -649,61 +649,61 @@ MPEG2VideoDecoderBase::~MPEG2VideoDecoderBase()
 
 Status MPEG2VideoDecoderBase::Reset()
 {
-    sequenceHeader.is_decoded = false;
-    sequenceHeader.first_i_occure  = 0;
-    sequenceHeader.first_p_occure  = 0;
-    sequenceHeader.broken_link = 0;
-    sequenceHeader.closed_gop  = 0;
-    sequenceHeader.time_code.gop_picture = 0;
-    sequenceHeader.time_code.gop_seconds = 0;
-    sequenceHeader.time_code.gop_minutes = 0;
-    sequenceHeader.time_code.gop_hours   = 0;
+    sequenceHeader.is_decoded                    = false;
```

Comment:
These kind of change, while beautiful, significantly complicate cherry picking across branches. I would not do them at all and focused on real changes. You have few of them below.

So, I strongly suggest to do beautification changes only across +/- 2-3 lines around the place you really changed.

Also, I personally don't like this particular change with aligning =. I prefer to have just single space around it otherwise you will have troubles adding/deleting members - you will be forced to realign the padding.

Reply:
@dvrogozh ok, I will remove the unecessary difference